pkg/runtime/compstore/compstore.go

```diff
@@ -35,41 +36,43 @@ import (
 type ComponentStore struct {
 	lock sync.RWMutex
 
-	states                  map[string]state.Store
-	configurations          map[string]configuration.Store
-	configurationSubscribes map[string]chan struct{}
-	secretsConfigurations   map[string]config.SecretsScope
-	secrets                 map[string]secretstores.SecretStore
-	inputBindings           map[string]bindings.InputBinding
-	inputBindingRoutes      map[string]string
-	outputBindings          map[string]bindings.OutputBinding
-	locks                   map[string]lock.Store
-	pubSubs                 map[string]PubsubItem
-	topicRoutes             map[string]TopicRoutes
-	workflowComponents      map[string]workflows.Workflow
-	cryptoProviders         map[string]crypto.SubtleCrypto
-	components              []compsv1alpha1.Component
-	subscriptions           []rtpubsub.Subscription
-	httpEndpoints           []httpEndpointV1alpha1.HTTPEndpoint
+	states                    map[string]state.Store
+	configurations            map[string]configuration.Store
+	configurationSubscribes   map[string]chan struct{}
+	secretsConfigurations     map[string]config.SecretsScope
+	secrets                   map[string]secretstores.SecretStore
+	inputBindings             map[string]bindings.InputBinding
+	inputBindingRoutes        map[string]string
+	outputBindings            map[string]bindings.OutputBinding
+	locks                     map[string]lock.Store
+	pubSubs                   map[string]PubsubItem
+	topicRoutes               map[string]TopicRoutes
+	workflowComponents        map[string]workflows.Workflow
+	workflowBackendComponents map[string]wfbe.WorkflowBackend
```

Comment:
Nit (not required): consider renaming this to `workflowBackend` since other component stores don't say "components"

Reply:
Go it, I followed workflow engine components just above this :p